etc/sysconfig/init

```diff
@@ -0,0 +1,8 @@
+# Common settings for normal and recovery init.
+
+# Amount of space to keep unallocated for refilling root or home later on.
+# Please keep this multiple of 500MB
+RESERVE_MB=0
```

Comment:
You would always want a certain minimum reserved as a full filesystem might cause issues on so many levels, logging broken, not being able to log in , ...